suite-native/module-accounts-import/src/components/SelectableNetworkList.tsx

```diff
@@ -32,10 +32,21 @@ const NetworkItemSection = ({
 );
 
 export const SelectableNetworkList = ({ onSelectItem }: SelectableAssetListProps) => {
-    const mainnetSymbols = getNativeMainnetSymbols();
-    const testnetSymbols = useSelector(selectSupportedTestnetNetworkSymbols);
+    const symbols = useSelector(selectDiscoveryNetworkSymbols);
     const areTestnetsEnabled = useSelector(selectAreTestnetsEnabled);
 
+    const [mainnetSymbols, testnetSymbols] = useMemo(
+        () =>
+            symbols.reduce<[NetworkSymbol[], NetworkSymbol[]]>(
+                ([mainnet, testnet], symbol) =>
+                    isTestnet(symbol)
+                        ? [mainnet, [...testnet, symbol]]
+                        : [[...mainnet, symbol], testnet],
+                [[], []],
+            ),
+        [symbols],
+    );
```

Comment:
Simplification :slightly_smiling_face: 
`import { arrayPartition } from '@trezor/utils';`

```suggestion
    const [testnetSymbols, mainnetSymbols] = useMemo(
        () => arrayPartition(symbols, isTestnet),
        [symbols],
    );
```

Reply:
updated/, squashing